### ISSUE TITLE
The display name of the calendar

### ISSUE THREAD

**Issue by grimmaldus** [grimmaldus/chorvatan-garbage#3]
**Is your feature request related to a problem? Please describe.**
Imported calendars to various clients shows either URL or ics file name instead of proper calendar name.

**Describe the solution you'd like**
Implement [iCalendar extension](https://en.wikipedia.org/wiki/ICalendar) `X-WR-CALNAME`

**Describe alternatives you've considered**
Implement other [iCalendar extension](https://en.wikipedia.org/wiki/ICalendar) extensions:

- `X-WR-CALDESC`
- `X-WR-TIMEZONE`
- `X-PUBLISHED-TTL`

**Additional context**
Google Calendar Client:
![Google Calendar From URL option](https://user-images.githubusercontent.com/9221276/150889730-926175c8-a921-468b-be56-9dffbde37363.JPG)

**Comment by dmnks** [comment]
Good point, you can always rename it manually afterwards, but it's better to make use of that extension if it's available :)

**Comment by dmnks** [comment]
>     * `X-WR-CALDESC`
>     * `X-WR-TIMEZONE`
>     * `X-PUBLISHED-TTL`

Are these also used for event description, though? It doesn't seem so.
